docs/howto/account-billing/change-billing-data.md

```diff
@@ -19,3 +19,30 @@ Finalize your changes by clicking the green _Update_ button at the bottom.
 Please note that you cannot readily change your organization number (for business accounts) or your personal number (for individual accounts) in the {{gui}}.
 
 In case you do wish to change any of those, you will have to submit a [{{legal_docs.transfer_form.name}}]({{legal_docs.transfer_form.url}}) via our [{{support}}](https://{{support_domain}}/servicedesk).
+
+## E-invoicing for Swedish customers
+
+This is available to select accounts.
+More specifically, you have the option of e-invoicing if all three conditions are met:
+
+* the country in the customer contract is set to _Sweden_ (that is the country in the account owner data, not the country in the billing data),
+* you have provided a company name,
+* you have provided a Swedish [organisationsnummer](https://www.bolagsverket.se/foretag/organisationsnummer.1207.html) (link available in Swedish only).
```

Comment:
I would put the non-English term in italics, and shorten the parenthetical expression:
```suggestion
* you have provided a Swedish *[organisationsnummer](https://www.bolagsverket.se/foretag/organisationsnummer.1207.html)* (link in Swedish).
```